src/CMakeLists.txt

```diff
@@ -151,6 +171,21 @@ if (BUILD_MODULES)
     add_subdirectory(modules)
 endif()
 
+# Include External projects with tests
+if (BUILD_TELEMETRY)
+    ExternalProject_Get_Property(telemetry BINARY_DIR)
+    set(TELEMETRY_TESTS_DIR ${BINARY_DIR}/lib/tests)
+    configure_file(
+        ${CMAKE_SOURCE_DIR}/cmake/ExternalTests.cmake.in
+        ${CMAKE_BINARY_DIR}/ExternalTests.cmake
+        @ONLY
+    )
```

Comment:
>) [](http://example.com/codeflow?start=4&length=1)

This ) here and the one on line 186 are still in the opposite style of the project's style.

Do:

foo)

Instead:

foo
)